### ISSUE TITLE
sql: make it easier to inspect keys

### ISSUE THREAD

**Issue by awoods187** [cockroachdb/cockroach#61625]
In 21.1, we've added capabilities to identify contended keys. In https://github.com/cockroachdb/cockroach/issues/61507 we discuss making views of contended keys, but even with that information, it can be hard to parse the keys:
```SQL
 SELECT * from tpcc.contended_keys;
  database_name | schema_name | name |                         key                          | num_contention_events
----------------+-------------+------+------------------------------------------------------+------------------------
  system        | public      | jobs | \227\211\375\010\337\316\243"\223\200\001\210        |                   127
  system        | public      | jobs | \227\211\375\010\337\316\243"\311\200\001\210        |                   127
  system        | public      | jobs | \227\211\375\010\337\316\243+\331\000\001\210        |                   127
  system        | public      | jobs | \227\211\375\010\337\316\243,\015\000\001\210        |                   127
  system        | public      | jobs | \227\211\375\010\337\316\243\030\354\000\001\212\211 |                   127
```
We need to make a function that makes it easy to parse these keys. 

The crdb_internal.pretty_key is not currently super helpful either:
```SQL
 SELECT DISTINCT
         database_name,
         schema_name,
         name,
         crdb_internal.pretty_key(key, 0),
         num_contention_events
    FROM crdb_internal.cluster_contention_events,
         crdb_internal.tables,
         crdb_internal.table_indexes
   WHERE crdb_internal.cluster_contention_events.index_id
         = crdb_internal.table_indexes.index_id
     AND crdb_internal.cluster_contention_events.table_id
         = crdb_internal.tables.table_id
ORDER BY num_contention_events DESC;
  database_name | schema_name |    name     |                     crdb_internal.pretty_key                     | num_contention_events
----------------+-------------+-------------+------------------------------------------------------------------+------------------------
  system        | public      | jobs        | /15/1/639461742496350209/0                                       |                   127
  system        | public      | jobs        | /15/1/639456872164982785/0                                       |                   127
  system        | public      | jobs        | /15/1/639456872168521729/0                                       |                   127
  system        | public      | jobs        | /15/1/639456872320532481/0                                       |                   127
  system        | public      | jobs        | /15/1/639456872323940353/0                                       |                   127
  system        | public      | jobs        | /15/1/639456872480407553/0                                       |                   127
```

Epic CRDB-6567

Jira issue: CRDB-2979

**Comment by jordanlewis** [comment]
@kevin-v-ngo to take a look at this for prioritization.

**Comment by awoods187** [comment]
Here's another look at the output:
```SQL
  database_name | schema_name |    name     |                    crdb_internal.pretty_key                     | num_contention_events
----------------+-------------+-------------+-----------------------------------------------------------------+------------------------
  system        | public      | jobs        | /15/1/648532460647546881/1/1                                    |                    16
  system        | public      | jobs        | /15/1/648532460647776257/1/1                                    |                    16
  system        | public      | jobs        | /15/1/648532460647809025/1/1                                    |                    16
  system        | public      | jobs        | /15/1/648532460647841793/1/1                                    |                    16
  system        | public      | jobs        | /15/1/648532460647874561/1/1                                    |                    16
  system        | public      | jobs        | /15/1/648532460647907329/1/1                                    |                    16
  system        | public      | jobs        | /15/1/648532460647940097/1/1                                    |                    16
  system        | public      | jobs        | /15/1/648532460648136705/1/1                                    |                    16
  system        | public      | jobs        | /15/1/648532460648235009/1/1                                    |                    16
  system        | public      | jobs        | /15/1/648534628228366337/0                                      |                    16
  system        | public      | jobs        | /15/1/648534806389555201/0                                      |                    16
  system        | public      | sqlliveness | /39/1/"ym\xa6\xd5\xf8\xc8C\xf1\xb2\xf5\f\xfb\xfe\x17\xbd\x99"/0 |                     1
  tpcc          | public      | warehouse   | /58/1/218/0                                                     |                     1
  tpcc          | public      | district    | /57/1/592/9/0                                                   |                     1
```
I can tell that it's table 58 index one, but I can't tell what `218/0` means. How would a developer take action based on know that the key `58/1/218/0 ` is experiencing contention?
